### PR TITLE
updated credentials parameter to client id

### PR DESCRIPTION
**Pull request by jochman**
Ready

## Description
- Changed `Credentials` parameter name to `Client ID` to match Cisco AMP API.
- Added detailed instructions

## Does it break backward compatibility?
   - No

## Must have
- [X] Tests (no test, no instance)
- [ ] Code Review

### CODE REVIEW

**Comment by jochman** [comment]
Will not pass circle - JS integration can't be package.
@kirbles19 please review `detaileddescription`

File: Integrations/integration-AMP.yml [hunk 1]
```diff
@@ -6,13 +6,15 @@ display: Cisco AMP
...
 description: Uses CISCO AMP Endpoint
+detaileddescription: Client ID and API Key (Password parameter in Demisto) can be
+  generated with the instruction [here]( https://www.cisco.com/c/en/us/support/docs/security/amp-endpoints/201121-Overview-of-the-Cisco-AMP-for-Endpoints.html#anc1).
 configuration:
 - display: Server URL (e.g. https://192.168.0.1)
   name: server
   defaultvalue: ""
```
Comment (Itay4) [inline Integrations/integration-AMP.yml]: add default value here

**Comment by kirbles19** [comment]
@jochman please provide a link to the exact file I'm supposed to review.

**Comment by jochman** [comment]
> @jochman please provide a link to the exact file I'm supposed to review.

https://github.com/demisto/content/pull/3898/files#diff-cf49d3bff2d56b86e6542112b9232272

**Pull request merged**